featurebyte/routes/static_source_table/api.py

```diff
@@ -28,6 +29,24 @@
 from featurebyte.schema.static_source_table import StaticSourceTableCreate, StaticSourceTableList
 from featurebyte.schema.task import Task
 
+
+class StaticSourceTableRouter(BaseMaterializedTableRouter[StaticSourceTableModel]):
+    """
+    Static source table router
+    """
+
+    table_model = StaticSourceTableModel
+    controller = "static_source_table_controller"
+
+    def __init__(self, prefix: str):
+        super().__init__(prefix=prefix)
+
+    async def get_table(
+        self, request: Request, static_source_table_id: PydanticObjectId
```

Comment:
override the method here to change the variable name, so that it matches the existing method variable names for backwards compatibility.